quiche/src/build.rs

```diff
@@ -157,18 +155,6 @@ fn get_boringssl_cmake_config() -> cmake::Config {
             _ => boringssl_cmake,
         },
 
-        "windows" => match arch.as_ref() {
-            "x86_64" => {
-                // Override _WIN32_WINNT to use 64bit APIs,
-                // such as GetTickCount64().
-                boringssl_cmake.cxxflag("-D_WIN32_WINNT=0x0600");
```

Comment:
Is it because we use `winlibs` which already defined `_WIN32_WINNT`?
I'd like to understand this is `winlibs` specific fix or not.

Reply:
Actually, ["egor-tensin/setup-mingw" installs mingw from chocolatey](https://github.com/egor-tensin/setup-mingw/blob/master/action.yml#L79-L82), which [installs mingw from WinLibs](https://community.chocolatey.org/packages/mingw#files). This is also the case for [mingw in the Windows CI images](https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Installers/Install-Mingw64.ps1). Therefore, this issue is unrelated to WinLibs.

I choose to use ["bwoodsend/setup-winlibs-action"](https://github.com/bwoodsend/setup-winlibs-action) due to more mordern compiler and library with better flexibility, e.g. MSVCRT runtime, UCRT runtime, and Clang compiler. Prior to it, ["msys2/setup-msys2"](https://github.com/msys2/setup-msys2)
 was under my consideration, but I could not find a way to use ["actions-rs/cargo"](https://github.com/actions-rs/cargo) in the msys2 shell.